2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -86,8 +86,12 @@ function _wait_for_mongo() {
 # To get list of endpoints, you need to have headless Service named 'mongodb'.
 # NOTE: This won't work with standalone Docker container.
 function endpoints() {
-  service_name=${MONGODB_SERVICE_NAME:-mongodb}
-  dig ${service_name} A +search +short 2>/dev/null
+  if [ -n "${MONGODB_REPLICA_MEMBERS+x}" ]; then
```

Comment:
I agree that `:-` also cryptic but it's used more often, looks more idiomatic and I don't need to open `man bash` again to find what `+` means in this context.